workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs

```diff
@@ -16,38 +17,29 @@ public static void AddLoadBalancingSystems(this WorkerInWorld worker, Action<Loa
 
     public class LoadBalancerConfiguration
     {
-        private readonly World world;
+        public readonly World World;
 
         internal LoadBalancerConfiguration(World world)
         {
-            this.world = world;
-            world.GetOrCreateSystem<ClassifyWorkersSystem>();
+            World = world;
+            World.GetOrCreateSystem<ClassifyWorkersSystem>();
         }
 
-        public void AddPartitionManagement(params string[] workerTypes)
+        public void AddPartitionManagement(string workerType, params string[] workerTypes)
         {
-            var partitionManagementSystem = world.GetOrCreateSystem<PartitionManagementSystem>();
-            partitionManagementSystem.WorkerTypes = workerTypes;
+            var partitionManagementSystem = World.GetOrCreateSystem<PartitionManagementSystem>();
+            partitionManagementSystem.WorkerTypes = workerTypes.Append(workerType).ToArray();
```

Comment:
If we pass in no additional `workerTypes`, is it null or an empty array?

Reply:
[Empty array](https://stackoverflow.com/questions/2942050/passing-zero-arguments-as-params-where-the-behaviour-is-defined)